client/http/types.go

```diff
@@ -592,3 +592,12 @@ type MembersInfo struct {
 	Leader     *pdpb.Member         `json:"leader,omitempty"`
 	EtcdLeader *pdpb.Member         `json:"etcd_leader,omitempty"`
 }
+
+// MicroServiceMember is the member info of a micro service.
+type MicroServiceMember struct {
```

Comment:
ServiceRegistryEntry and MicroServiceMember should be always the same? We need to add some comments to avoid inconsistent after late modification.

Reply:
unify their name